application.py

```diff
@@ -335,24 +335,24 @@ def add_bank_details():
     'intro': "I'm TheCoronials bot for COVID-19 Hackathon!",
```

Comment:
I would change the sentence here to make it clear we want the type of information the user want.

Maybe:
- Are you ...
-- an individual
-- a Business

To make it more natural